core/agent/agentinfo.go

```diff
@@ -1,19 +1,16 @@
 // Copyright 2015 Canonical Ltd.
 // Licensed under the AGPLv3, see LICENCE file for details.
 
-package service
+package agent
```

Comment:
This should be moved to the top level juju/agent package.
The stuff it contains all pertains to logic in that package